console/lib/auth_service.js

```diff
@@ -15,14 +15,20 @@
  */
 'use strict';
 
+var path = require('path');
+var fs = require('fs');
 var rhea = require('rhea');
 var Promise = require('bluebird');
+var ca_dir = "/opt/console/authservice-ca";
 
 function authenticate(user) {
     return new Promise(function(resolve, reject) {
         var options = {
-            "host": process.env.AUTHENTICATION_SERVICE_HOST,
-            "port": process.env.AUTHENTICATION_SERVICE_PORT
+            host: process.env.AUTHENTICATION_SERVICE_HOST,
+            port: process.env.AUTHENTICATION_SERVICE_PORT,
+            transport: 'tls',
+            ca: [ fs.readFileSync(path.resolve(ca_dir, 'tls.crt')) ],
```

Comment:
Possibly worth a try/catch around the reading of the cert to make it obvious is there is any problem there(?)